sdk/core/azure-core/src/main/java/com/azure/core/http/policy/DefaultRedirectStrategy.java

```diff
@@ -73,15 +73,14 @@ public DefaultRedirectStrategy(int maxAttempts, String locationHeader, Set<HttpM
         }
         this.maxAttempts = maxAttempts;
         if (CoreUtils.isNullOrEmpty(locationHeader)) {
-            logger.error(String.format("'locationHeader' provided as null will be defaulted to %s",
-                DEFAULT_REDIRECT_LOCATION_HEADER_NAME));
+            logger.error(
+                "'locationHeader' provided as null will be defaulted to " + DEFAULT_REDIRECT_LOCATION_HEADER_NAME);
```

Comment:
This is even worse! :) Don't do string concat, just use the var-args method to pass in the arguments like elsewhere in this code.

Reply:
updated!